src/schema/v2/homeView/HomeViewSection.ts

```diff
@@ -237,8 +237,19 @@ const SalesRailHomeViewSectionType = new GraphQLObjectType<
   },
 })
 
-// the Section union type of all concrete sections
+export const GalleriesHomeViewSectionType = new GraphQLObjectType<
```

Comment:
**thought (non-blocking):** 🤦🏽 sorry for this late feedback. I think I was expecting to see a rail of galleries here, but now I realize of course this is just a big hero image that navigates to a predefined route. 

This seems like the kind of section we _could_ handle generically (HeroImageLink or some such). 

I don't think it's a requirement to do that now, since this is just a one-off in the current home feed design. But should we? A little hard to say without seeing more complete designs. If this pattern is recurring it would make sense to treat it generically. Also that would free up the name GalleriesHomeViewSection for something that really _was_ a list of galleries.

Like I said, nothing urgent, since we are still free to make breaking changes till M2.

Reply:
Could we cover >1 section by making this generic? Like, could it reuse the hero section, with different parameters? If so, seems like a win.

Either way, I imagine we _will_ need a general-purpose type of section for static content. So sooner or later...

Reply:
Followed up here finally, and worth some discussion: https://github.com/artsy/metaphysics/pull/6116/